functionality/database/src/main/kotlin/pl/patrykgoworowski/mintlift/functionality/database/di/DatabaseDateFormat.kt

```diff
@@ -0,0 +1,6 @@
+package pl.patrykgoworowski.mintlift.functionality.database.di
```

Comment:
```suggestion
package com.patrykandpatryk.liftapp.functionality.database.di
```

I’d also suggest renaming the packages accordingly.